packages/quantic/force-app/main/default/lwc/quanticSvg/quanticSvg.js

```diff
@@ -0,0 +1,23 @@
+import { LightningElement, api } from 'lwc';
```

Comment:
See the comments from [this PR](https://github.com/coveo/ui-kit/pull/990) for this file.